packages/ecc-utils-design/src/components/details/details.ts

```diff
@@ -269,7 +209,8 @@ export default class EccUtilsDesignDetails extends LitElement {
           ${field.label}
           ${field.copy
             ? html`<sl-copy-button
-                .value=${_.get(this.data, field.path)}
+                style="background-color: #f5f5f5; border: 1px solid #e0e0e0; color: #333;"
```

Comment:
is there a reason why this needs inline style?

Reply:
Nope I just missed this! Thanks